eng/Version.Details.xml

```diff
@@ -1,12 +1,12 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="8.0.100-rtm.23530.12">
+    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="8.0.101-servicing.23614.6">
       <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>ae28d5bc061eedfe863ef3e22d090054b69e7685</Sha>
+      <Sha>461c7766428ba2782725d89d95d3f4f414ff8c92</Sha>
     </Dependency>
```

Comment:
@rmarinho if the goal here is to update `xharness`, why are we also updating the .NET SDK? Using a nightly `8.0.101-servicing.23614.6` SDK would require some nightly feeds, for the unreleased 7.0.17 runtime.

Reply:
Because xharness is trying ro find a 8.0.0 version and rtm is lower. 
Can we add  the feeds ?

Reply:
Or do we havea previous version for 7.0.16 ?

Reply:
> Because xharness is trying ro find a 8.0.0 version and rtm is lower.

What does that mean? Can you mention this in the PR description/commit message? Was there an error message before?

Reply:
7.0.16 was released last week: https://www.nuget.org/packages/Microsoft.NETCore.App.Runtime.Mono.android-arm/7.0.16